backend/app/Models/Transaction.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Transaction extends Model
+{
+    const FIELD_TRANSACTION_TIME = 'transaction_time';
+
+    public function transactionCard()
+    {
+        return $this->BelongsTo(Card::class, 'id', 'card_id');
```

Comment:
`Card::class` probably doesn't resolve here, as it isn't imported. Instead use `'App\Models\Card'`